src/test/java/org/jitsi/meet/test/MuteTest.java

```diff
@@ -159,29 +159,25 @@ public void ownerMutesParticipantAndCheck()
 
         WebElement elem = owner.findElement(By.xpath(
             TestUtils.getXPathStringForClassName("//span", "remotevideomenu")
-            + "/i[@class='icon-thumb-menu']"));
+            + "//i[@class='icon-thumb-menu']"));
```

Comment:
Yes, it's worthwhile but almost all these files have these selectors inlined. What needs to happen, and what I'd really like to do this year, is start moving these over into page objects to encapsulate these lookups, moving these to constants in the process.